source/decryptor/decryptor.c

```diff
@@ -99,54 +100,52 @@ u32 DecryptTitlekeys(void)
 
 u32 NcchPadgen()
 {
-    size_t bytesRead;
     u32 result;
 
     NcchInfo *info = (NcchInfo*)0x20316000;
     SeedInfo *seedinfo = (SeedInfo*)0x20400000;
 
-    if (FileOpen("/slot0x25KeyX.bin")) {
+    if (DebugFileOpen("/slot0x25KeyX.bin")) {
         u8 slot0x25KeyX[16] = {0};
-        Debug("Opening slot0x25KeyX.bin ...");
-
-        bytesRead = FileRead(&slot0x25KeyX, 16, 0);
-        FileClose();
-        if (bytesRead != 16) {
-            Debug("slot0x25KeyX.bin is too small!");
+        if (!DebugFileRead(&slot0x25KeyX, 16, 0))
             return 1;
-        }
+        FileClose();
         setup_aeskeyX(0x25, slot0x25KeyX);
     } else {
-        Debug("Warning, not using slot0x25KeyX.bin");
+        // Debug("Warning, not using slot0x25KeyX.bin");
```

Comment:
Because it is redundant information. Because you're using DebugFileOpen(), you first get: "Could not open slot0x25KeyX.bin!"
then (without commenting out):
"Warning, not using slot0x25KeyX.bin"
The last commit removes this line, but if you want to still have it, let me know.